sympy/series/tests/test_limits.py

```diff
@@ -880,6 +880,14 @@ def test_issue_18118():
     assert limit(sign(sin(x)), x, 0, "-") == -1
     assert limit(sign(sin(x)), x, 0, "+") == 1
 
+def test_issue_18176():
```

Comment:
These seem quite similar to #22230. Perhaps both of these can be mergeg and put into another appropriate test function (like `test_exponential`) rather than making new functions.

Reply:
Thanks @0sidharth, I had completely forgotten about the earlier Pr . Do we even need this Pr then both address similar issues !